docs/tutorial/workers.md

```diff
@@ -17,13 +17,13 @@ search:
 
 ## Why workers and work pools?
 
-Workers and work pools bridge the Prefect orchestration layer with the infrastructure the flows are actually executed on. 
+Workers and work pools are a bridge between the Prefect orchestration layer and infrastructure that enable flow runs to be executed in dynamically provisioned environments. To transition from persistent infrastructure to dynamic infrastructure, you can use `flow.deploy` instead of `flow.serve`.
```

Comment:
```suggestion
Workers and work pools are a bridge between the Prefect orchestration layer and infrastructure for flow runs that can be dynamically provisioned. To transition from persistent infrastructure to dynamic infrastructure, use `flow.deploy` instead of `flow.serve`.
```